theblues/charmstore.py

```diff
@@ -361,6 +401,10 @@ def _common_query_parameters(self, doc_type, includes, owner,
         return queries
 
     def fetch_related(self, ids):
+        """Fetch related entity information.
+
+        @param ids The entity ids to fetch related information for.
```

Comment:
... like stats, extra information, bundle metadata, and unit count for bundle identifiers, supported series for charms...
or similar?